common/src/main/java/com/wynntils/utils/mc/type/Location.java

```diff
@@ -71,10 +70,6 @@ public Vec3 toVec3() {
         return new Vec3(x, y, z);
     }
 
-    public Position toPosition() {
```

Comment:
I'm a bit surprised this was not used..? Don't you think we should keep it but return Vec3 or whatever it is they use nowadays for a basic implementation of Position?

Reply:
We could but if the underlying class is Vec3 you can implicitly cast it

Reply:
I still think I'd like to see this kept; there is a difference in semantics that I'd like to keep, where Position is the more abstract Minecraft concept of a general (entity) position, basically a location with fractional coordinates.